docs/content/configuration/index.md

```diff
@@ -803,9 +803,9 @@ Issuing a GET request at the same URL will return the spec that is currently in
-|`historicalNodesInMaintenance`| List of Historical nodes in maintenance mode. Coordinator doesn't assign new segments on those nodes and moves segments from the nodes according to a specified priority.|none|
-|`nodesInMaintenancePriority`| Priority of segments from servers in maintenance. Coordinator takes ceil(maxSegmentsToMove * (priority / 10)) from servers in maitenance during balancing phase, i.e.:<br>0 - no segments from servers in maintenance will be processed during balancing<br>5 - 50% segments from servers in maintenance<br>10 - 100% segments from servers in maintenance<br>By leveraging the priority an operator can prevent general nodes from overload or decrease maitenance time instead.|7|
+|`maxSegmentsInNodeLoadingQueue`|The maximum number of segments that could be queued for loading to any given server. This parameter could be used to speed up segments loading process, especially if there are "slow" nodes in the cluster (with low loading speed) or if too much segments scheduled to be replicated to some particular node (faster loading could be preferred to better segments distribution). Desired value depends on segments loading speed, acceptable replication time and number of nodes. Value 1000 could be a start point for a rather big cluster. Default value is 0 (loading queue is unbounded) |0|
+|`decommissioningNodes`| List of historical servers to 'decommission'. Coordinator will not assign new segments to 'decomissioning' servers,  and segments will be moved away from them to be placed on 'active' servers at the maximum rate specified by `decommissioningMaxPercentOfMaxSegmentsToMove`.|none|
+|`decommissioningMaxPercentOfMaxSegmentsToMove`|  The maximum number of segments that may be moved away from 'decommissioning' servers to non-decommissioning (that is, active) servers during one Coordinator's run. This value is relative to the total maximum segment movements allowed during one run which is determined by `maxSegmentsToMove`. If `decommissioningMaxPercentOfMaxSegmentsToMove` is 0, segments will neither be moved from _or to_ 'decommissioning' servers, effectively putting them in a sort of 'maintenance' mode that will not participate in balancing or assignment by load rules. Decommissioning can also become stalled if there are no available active servers to place the segments. By leveraging decommissioning percent, an operator can prevent active servers from overload by prioritizing balancing, or decrease decommissioning time instead. The value should be between 0 and 100.|70|
```

Comment:
I suggest the following (a few fixes highlighted in italics):
> *Determines* the maximum number of segments that may be moved away from 'decommissioning' servers to non-decommissioning (that is, active) servers during one Coordinator's run. *The maximum number of segment movements away from 'decommissioning' servers* is relative to the total maximum segment movements allowed during one run which is determined by `maxSegmentsToMove`. If `decommissioningMaxPercentOfMaxSegmentsToMove` is 0, segments will neither be moved from _or to_ 'decommissioning' servers, effectively putting them in a sort of *"maintenance"* mode that will not participate in balancing or assignment by load rules. Decommissioning can also become stalled if there are no available active servers to place the segments. By leveraging *the maximum percent of decommissioning segment movements*, an operator can prevent active servers from overload by prioritizing balancing, or decrease decommissioning time instead. The value should be between 0 and 100.

Reply:
I think starting with `The maximum number` is pretty consistent with the other descriptions:

```
|`mergeBytesLimit`|The maximum total uncompressed size in bytes ...
|`mergeSegmentsLimit`|The maximum number of segments that can be ...
|`maxSegmentsToMove`|The maximum number of segments that can be ...
|`replicantLifetime`|The maximum number of Coordinator runs for ...
|`replicationThrottleLimit`|The maximum number of segments that ...
```

I also think that repeating the near same phrase in the 2nd sentence isn't really necessary, but agree with the other changes :+1:

Reply:
I don't see why this kind of "consistency" at the beginnings of the doc descriptions is a good thing.

You can reword it to start with "The maximum percent", but then the whole sentence should be changed.

The problem with "This value" is that *the percent value* is not relative to `maxSegmentsToMove`. *The result of applying the percent* to `maxSegmentsToMove` is relative to `maxSegmentsToMove` (quite obviously).

Reply:
Hmm, yeah i see the problem, I think some things can be reworded. How about this, where I sort of combine the ideas of the first 2 sentences, and also throw in a reference to `decommissioningNodes`, maybe:

> The percent of `maxSegmentsToMove` that determines the maximum number of segments that may be moved away from 'decommissioning' servers (specified by `decommissioningNodes`) to non-decommissioning servers during one Coordinator balancer run. If `decommissioningMaxPercentOfMaxSegmentsToMove` is 0, segments will neither be moved from or to 'decommissioning' servers, effectively putting them in a sort of "maintenance" mode that will not participate in balancing or assignment by load rules. Decommissioning can also become stalled if there are no available active servers to place the segments. By adjusting `decommissioningMaxPercentOfMaxSegmentsToMove`, an operator can prevent active servers from overload by prioritizing balancing, or decrease decommissioning time instead. The value should be between 0 and 100.

Reply:
@clintropolis's proposed wording looks good to me.

Reply:
👍 good wording.